app/src/main/java/app/revanced/integrations/youtube/patches/misc/requests/StreamingDataRequest.java

```diff
@@ -228,18 +242,33 @@ private static ByteBuffer fetch(@NonNull String videoId, Map<String, String> pla
                 try {
                     // gzip encoding doesn't response with content length (-1),
                     // but empty response body does.
-                    if (connection.getContentLength() != 0) {
-                        try (InputStream inputStream = new BufferedInputStream(connection.getInputStream())) {
-                            try (ByteArrayOutputStream baos = new ByteArrayOutputStream()) {
-                                byte[] buffer = new byte[8192];
-                                int bytesRead;
-                                while ((bytesRead = inputStream.read(buffer)) >= 0) {
-                                    baos.write(buffer, 0, bytesRead);
+                    int contentLength = connection.getContentLength();
```

Comment:
When I tested the PR, this code is not reachable in this live stream
(not executed)

Reply:
Uhm as I described in the PRs, this check will only apply when the IOS Compability Mode is disabled

Reply:
Even with iOS compatibility mode turned off, it is still spoofed as an Android VR client or Android TV client in the livestream, not as an iOS client

This does not match the summary:
```
<string name="revanced_spoof_streaming_data_ios_compatibility_summary_off">Always spoofed as iOS client.</string>
```

Reply:
Ah my mistake, I will fix it later